src/IFrameManager.js

```diff
@@ -105,6 +105,9 @@ class IFrameManager {
     this.onMessageHandler({
       type: messageType.SYNC_ACK_FAILED,
     });
+    this.sendMessage({
+      type: messageType.SYNC_ACK_FAILED,
+    });
```

Comment:
Is the iframe.onerror event related to the SYNC event from HubSpot? If not, we should probably rename the messageType because the naming makes it seem like it is.

Reply:
If HubSpot doesn't receive the `SYNC_ACK` event after 30s, it creates a SYNC_ACK_FAILED message and sends it to the EmbeddableComponentClient. It wouldn't be able to send the SYNC_ACK_FAILED message to the SDK since the SDK did not load.